tombstone_gc.hh

```diff
@@ -28,23 +30,36 @@ class database;
 
 }
 
-class repair_history_map;
-using per_table_history_maps = std::unordered_map<table_id, seastar::lw_shared_ptr<repair_history_map>>;
+using reconcile_history_map = boost::icl::interval_map<dht::token, gc_clock::time_point, boost::icl::partial_absorber, std::less, boost::icl::inplace_max>;
+
+class per_table_history_maps {
+public:
+    std::unordered_map<table_id, seastar::lw_shared_ptr<reconcile_history_map>> _repair_maps;
+    seastar::lw_shared_ptr<reconcile_history_map> _group0_map;
```

Comment:
Please add a comment why all group 0 tables are using a single map

Reply:
added the explanatory comment